app/js/models/tree.js

```diff
@@ -0,0 +1,20 @@
+var app = app || {};
+
+app.models = app.models || {};
+
+Backbone.Model.prototype.idAttribute = 'id';
```

Comment:
This overrides the idAttribute for **all** backbone models in the app. I think it's better to put this in the extend below:
```js
Backbone.Model.extend({
    idAttribute: 'id'
});
```